packages/datetime/test/common/dateFormatPropsTests.ts

```diff
@@ -0,0 +1,60 @@
+/* !
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ */
+
+import { expect } from "chai";
+import { format } from "date-fns";
+
+import { getFormattedDateString } from "../../src/common/dateFormatProps";
+import { Months } from "../../src/common/months";
+
+const formatDate = (date: Date) => format(date, "yyyy-MM-dd");
+
+describe("DateFormatProps", () => {
+    describe("getFormattedDateString", () => {
+        it("should return an empty string for null date", () => {
+            const testDate: Date | null = null;
+            const props = { formatDate };
```

Comment:
Nit: Here and below, add a type annotation for these prop object literals or inline them into the function call. Context: https://pages.github.palantir.build/palantir/documentation/#/foundry/develop/frontend/forge-style-guide?id=annotate-types-of-variables-assigned-from-object-literals